cloudinit/util.py

```diff
@@ -3073,6 +3073,14 @@ def wait_for_files(flist, maxwait, naplen=0.5, log_pre=""):
     return need
 
 
+def wait_for_snap_seeded():
+    """Helper to wait on completion of snap seeding."""
+    if not subp.which("snap"):
```

Comment:
Will Ubuntu ever not have `snap` installed? All of the callsites are ubuntu-only modules, so I'm curious why we feel the need for this check.

Reply:
I guess it won't hurt anything, I just assume that it is unnecessary

Reply:
Good point, while all ubuntu images at the present time all have snaps included or seeded in their images, this may change if the future for specific minimal images. This was just a preliminary pre-flight check to ensure we don't go down a path that would `raise CalledProcessError` on a missing command if we happen to be in an env that doesn't support snaps for some reason.